### ISSUE TITLE
Post member exceptions as comments on CRM issues

### ISSUE THREAD

**Issue by chadwhitacre** [opensourcepledge/opensourcepledge.com#241]
Currently with #240 they create new issues in the main repo. Not sure if we want a public ticket for each one of these but it would be great to at least link over to the relevant private CRM issue.

**Comment by vladh** [comment]
I guess a good reason to keep the issues in this repo would be that they would be visible to the members and to the community.

Could think about how to link them to the CRM repo though!

**Comment by chadwhitacre** [comment]
Fair enough. If it ever gets to feel like too much noise in this repo we can adjust.

I guess to link we would need to maintain a mapping of members to issue IDs or somesuch.

**Comment by vladh** [comment]
I think this feature wouldn't be helpful enough to justify the work, but I'm happy to do this if you still need it @chadwhitacre!
